caffe2/CMakeLists.txt

```diff
@@ -11,6 +11,13 @@ if(MSVC)
   include(../cmake/public/utils.cmake)
 endif()
 
+# Debug messages - if you want to get a list of source files and examine
+# target information enable the following by `set(ADD_DEBUG_INFO TRUE)`.
+set(ADD_DEBUG_INFO FALSE)
```

Comment:
you can do CACHE STRING or put this as a top-level option.

Reply:
Update it as an option. I'm thinking that it's mainly used for debugging CMake target, instead of other options, which are mostly used as configs. It's probably better to leave it in this level, so it's closer to the actual debug information.

Reply:
oops. sorry I misunderstood your use case. 

In this case I think you don't need an option. it is better to use.
```
set(PRINT_DEBUG_INFO FALSE CACHE BOOL "print cmake debug information")
```
This way `PRINT_DEBUG_INFO` will be available as a CACHE option in ccmake/cmake-gui and you can edit this in tools like vscode or directly in build/CMakeCache.txt

Reply:
Ah I see. Thanks for the detailed explanation. Update it in the latest commit.